### DISCUSSION TITLE
error "undefined" in product preview when selecting combinations

### DISCUSSION THREAD

**Discussion by arsonists** [PrestaShop/PrestaShop#35880] [Q&A]
Good afternoon, after trying to fix it by my means... I finally open this post here before reporting it as a bug.
I am in prestashop 8.1.5
My problem is that when I view a product with combinations disabled (preview) and select a combination, I get the error "undefined".

![error undefined](https://github.com/PrestaShop/PrestaShop/assets/22270830/0cd7a2f9-5eb5-4bb9-82fe-c095821d2801)

Looking at the server logs I only get the following error:

![error undefined 2](https://github.com/PrestaShop/PrestaShop/assets/22270830/ec1d0d19-6007-4459-9ef5-45e0dae431d0)

In debug mode I can't find any error... I've been investigating and I'm not the only one that happens to him, in the official forums there are people the same.

https://www.prestashop.com/forums/topic/1079833-after-selecting-product-attribute-value-in-inactive-showing-undefined-error/
https://www.prestashop.com/forums/topic/1080155-undefined-error/

Anyway, let's see if you can help me.

Best regards.

**Comment by matks** [comment]
Hello, even more helpful than the server logs, you have the PHP error logs. Can you check them?

**Comment by arsonists** [comment]
Good morning, first of all thank you for answering. I tell you, the only error I get in php is:

![Screenshot_10](https://github.com/PrestaShop/PrestaShop/assets/22270830/81e30f94-509a-4803-8c5b-6133a6d742f1)
